src/main/java/application/controllers/DevToolController.java

```diff
@@ -86,6 +90,27 @@ public void closeSurvey(HttpServletResponse response) throws IOException {
         List<Survey> list = Lists.newArrayList(surveys);
 
         WriteCsvToResponse.writeSurveys(response.getWriter(), list);
-        log.info("DevTool export function was triggered with %d surveys", list.size());
+        log.info("DevTool export function was triggered with {} surveys", list.size());
+    }
+
+    private final static List<String> VALID_LEVELS =  Lists.newArrayList("TRACE", "DEBUG", "INFO", "WARN", "ERROR");
+
+    /***
+     * Developer tool used to set the log level of this software package
+     * @param logLevel logback log level
+     * @return status message directly in response body
+     */
+    @GetMapping("/log_{logLevel}")
```

Comment:
I used this for the presentation to demonstration changing log level at runtime, but not sure if this is really needed.